.github/workflows/build.yml

```diff
@@ -17,20 +17,20 @@ jobs:
         graphql-version: [15, 16, 17.0.0-alpha.3]
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
 
       # https://github.com/actions/cache/blob/main/examples.md#node---yarn
       - name: Get yarn cache directory path
         id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
```

Comment:
set-output is deprecated. The new way is to use env vars https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#examples